ethereum-consensus/src/deneb/polynomial_commitments.rs

```diff
@@ -0,0 +1,152 @@
+use crate::{deneb::blob_sidecar::Blob, primitives::Bytes32, ssz::prelude::*};
+use c_kzg::Error;
+pub use c_kzg::KzgSettings;
+
+use super::mainnet::FIELD_ELEMENTS_PER_BLOB;
+
+pub const BYTES_PER_FIELD_ELEMENT: usize = 32;
+pub const BYTES_PER_BLOB: usize = BYTES_PER_FIELD_ELEMENT * FIELD_ELEMENTS_PER_BLOB;
```

Comment:
same here, this is a parameter of this module so we shouldn't define a concrete instantiation of it